### ISSUE TITLE
Signup Data Form: why_signedup JS validation

### ISSUE THREAD

**Issue by aaronschachter** [DoSomethingArchive/legacy-website#1917]
Blocked by #1912 

If the signup data form is configured to collect the `why_signedup` property, JS validation should check to make sure a value exists.

**Comment by mikefantini** [comment]
@barryclark shouldn't this be high priority? we need to make sure people give this information otherwise we won't be able to give them Thumb Socks and shit will just get messy.

Is this particular issue for client side vs. server side validation? Is server side already in place?

**Comment by aaronschachter** [comment]
Yes @mikefantini,  this particular issue is for client side -- server side is already in place.

**Comment by barryclark** [comment]
@mikefantini Yea, server side is done, that's why it's low priority.

**Comment by mikefantini** [comment]
cool makes sense
